Docs/source/How_To/coding-style.rst

```diff
@@ -0,0 +1,271 @@
+*************
+Coding Style
+*************
+
+.. note:: 
+
+    | CamelCase - FirstWordSecondWord
+    | camelCase - firstWordSecondWord
+
+Macros and Constants
+====================
+
+Should be written in snakecase with all caps. If a macro utilizes arithmetic, it should be covered in parentheses to prevent compiler
```

Comment:
```suggestion
Macros and constants should be written in snakecase with all caps. If a macro utilizes arithmetic, it should be covered in parentheses to prevent the compiler
```